README.md

```diff
@@ -16,6 +16,12 @@
 
 ### Tests lokal ausführen:
 
+#### Lokale Installation
+
+npm ci
+npx playwright install
```

Comment:
Ist diese Zeile notwendig? Das wird doch in der nächsten Zeile installiert, oder warum brauchen wir vorher npx playwright install ?